app/components/provider_interface/organisation_permissions_form_component.html.erb

```diff
@@ -14,12 +14,14 @@
 
   <% ProviderRelationshipPermissions::PERMISSIONS.each do |permission_name| %>
     <div class="govuk-form-group">
-      <%= f.govuk_fieldset legend: { text: label_for(permission_name) } do %>
-        <% presenter.checkbox_details_for_providers.each_with_index do |checkbox_details, index| %>
+      <%= f.govuk_check_boxes_fieldset permission_name, legend: { text: label_for(permission_name) } do %>
```

Comment:
This generates slightly weird form parameters, which I suppose is fine as long as we only permit the ones we want in the controller.

```
provider_relationship_permissions[make_decisions][]: 
provider_relationship_permissions[training_provider_can_make_decisions][]: training_provider_can_make_decisions
provider_relationship_permissions[ratifying_provider_can_make_decisions][]: ratifying_provider_can_make_decisions
provider_relationship_permissions[view_safeguarding_information][]: 
provider_relationship_permissions[training_provider_can_view_safeguarding_information][]: training_provider_can_view_safeguarding_information
```

Reply:
I think the check_boxes_fieldset/check_box is generating the empty `make_decisions` parameter